var a = [1,2,3];

try {
    console.log(a[i]);
} catch (error) {
    console.log("エラーが発生しました: " + error.message);
}